src/reducers/StakingReducer.ts

```diff
@@ -132,15 +125,6 @@ export const walletStakingStateReducer: Reducer<WalletStakingState, Action> = co
         return state
     }
   },
-  stakePlugins: (state = [], action: Action) => {
-    switch (action.type) {
-      case 'STAKING/UPDATE_PLUGINS':
```

Comment:
Ok, but delete the `STAKING/UPDATE_PLUGINS` action altogether, because there are no remaining reducers that listen for it.